data/wanderer/wanderers start.txt

```diff
@@ -2015,18 +2085,17 @@ mission "eastern evacuation"
 		fail
 
 mission "Wanderers Evacuation 1"
+	landing
 	name "Evacuate <planet>"
 	description `Escort another fleet of Wanderer refugee transports to <destination>.`
 	autosave
 	source "Vara Ke'sok"
 	destination "Vara Ke'stai"
 	to offer
-		or
-			has "Wanderers Solifuge Recon 2: done"
-			has "Wanderers Pond Strider Recon 2: done"
+		has "Wanderers Solifuge Recon 4: done"
 	on offer
 		conversation
-			`Four Wanderer ships that you've never seen before are parked here. These must be the dedicated transport ships that Iktat Rek mentioned. One of the captains approaches you and says, "We are evacuating the worlds that will be [isolated, trapped] if the Unfettered gain more territory. Will you escort us to <planet>, and from there to the capital?"`
+			`When you return to Wanderer space, you find four Wanderer ships that you've never seen before are parked near the spaceport. These must be the dedicated transport ships that Iktat Rek mentioned. One of the captains approaches you and says, "We are evacuating the worlds that will be [isolated, trapped] if the Unfettered gain more territory. Will you escort us to <planet>, and from there to the capital?"`
```

Comment:
While this change does make this chunk flow better from a literary point of view, remember that the first part "When you return to Wanderer Space" isn't something that needs to be said because the player is *doing* that. You are basically describing what the player has already done. Essential for a book, but not important when the player has just done that action in-game. 

As such, I don't think this change is necessary or beneficial.

Reply:
Well this form of transition is used elsewhere in the game, typically in the form `"When you return to <planet>"` or similar. As you say, it makes thing flow better from a literary standpoint, which is the whole reason to include a transition like this, bridging the gap between the gameplay side of things and the mission side of things.